packages/react-kit/src/components/form/Upload/BaseUpload.tsx

```diff
@@ -347,6 +385,28 @@ function BaseUpload({
                         muted
                         loop
                       />
+                    ) : isPdfOnly ? (
+                      <Grid
+                        flexDirection="row"
+                        alignItems="center"
+                        gap="0.25rem"
+                      >
+                        <div>
+                          <FilePdf size={24} />
+                        </div>
+                        <Typography>{preview}</Typography>
+                        <button
+                          type="button"
+                          style={{ display: "flex", justifyContent: "center" }}
+                          onClick={(e) => {
+                            e.preventDefault();
+                            e.stopPropagation();
+                            handleRemoveAllFiles();
+                          }}
+                        >
+                          <X size={12.5} />
+                        </button>
+                      </Grid>
```

Comment:
I think this should be themable/stylable, for example, what if in fermion we want this to have rounded edges but not in boson?

Reply:
It is container for content only, `FileUploadWrapper` is still responsible for input look